src/packet_analysis/Manager.cc

```diff
@@ -30,7 +30,10 @@ void Manager::InitPostScript(const std::string& unprocessed_output_file)
 	for ( const auto& analyzerComponent : GetComponents() )
 		{
 		if ( AnalyzerPtr newAnalyzer = InstantiateAnalyzer(analyzerComponent->Tag()) )
+			{
+			newAnalyzer->SetEnabled(analyzerComponent->Enabled());
```

Comment:
Only thought was if that could be moved into `InstantiateAnalyzer()` as the contract seems that one currently is required to explicitly call `SetEnabled()` on the result.

After looking, that method is private for the packet_analysis/Manager, so that seems mood currently.

Reply:
Yeah, what's why I moved that out of public.